server/routes/api/collections.ts

```diff
@@ -134,22 +142,45 @@ router.post("collections.info", auth(), async (ctx) => {
 });
 
 router.post("collections.import", auth(), async (ctx) => {
-  const { type, attachmentId } = ctx.body;
-  assertIn(type, ["outline"], "type must be one of 'outline'");
+  const { attachmentId } = ctx.body;
   assertUuid(attachmentId, "attachmentId is required");
+
   const { user } = ctx.state;
   authorize(user, "importCollection", user.team);
+
   const attachment = await Attachment.findByPk(attachmentId);
   authorize(user, "read", attachment);
-  await Event.create({
-    name: "collections.import",
-    modelId: attachmentId,
-    teamId: user.teamId,
-    actorId: user.id,
-    data: {
-      type,
-    },
-    ip: ctx.request.ip,
+
+  await sequelize.transaction(async (transaction) => {
+    const fileOperation = await FileOperation.create(
+      {
+        type: FileOperationType.Import,
+        state: FileOperationState.Creating,
+        format: FileOperationFormat.MarkdownZip,
```

Comment:
This will be passed in the request from the client once we have more format support